sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.JsonSchema/src/Microsoft.Azure.Data.SchemaRegistry.JsonSchema.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Does the naming here make sense given that we don't depend on any third party libraries. I wonder if this should just be Azure.Data.SchemaRegistry.JsonSchema.

Reply:
I think the only argument for keeping it is to align with the avro serializer

Reply:
Talked to Ted and am renaming for now. I'll also drop an email to Krzysztof to confirm.